api/tests/test_user_feedback_api.py

```diff
@@ -89,3 +89,19 @@ def test_can_get_all_feedback(self, mock_verify_token):
         self.assertEqual(response.status_code, 200)
         self.assertIn(response.data['results'][0]["message"],
                       "This is some feedback")
+
+    @patch('api.authentication.auth.verify_id_token')
+    def test_created_feature_request_not_resolved(self, mock_verify_token):
+        mock_verify_token.return_value = {'email': self.user.email}
+        data = {
+            "message": "Buy durable dongles",
+            "report_type": "feature request"
+        }
+        response = client.post(
+            self.feedback_url,
+            data=data,
+            HTTP_AUTHORIZATION="Token {}".format(self.token_user))
+
+        self.assertNotEqual(response.data, "resolved")
```

Comment:
🤔Of course `response.data` is not equal to "resolved". It is:
```python
{'reported_by': 'test4@site.com', 'message': 'Buy durable dongles', 'report_type': 'feature request', 'created_at': '2018-09-05T15:00:20.631259Z'}
```
You need to include the `resolved` field on serializer fields to check that its value is `false`.

Reply:
@collinmutembei done